### ISSUE TITLE
Big Exploit -> Win10 CTRL+Q duplicate items

### ISSUE THREAD

**Issue by sc00scoc** [GenisysPro/GenisysPro#701]
Hi, if I try to do ctrl + q with onDrops disabled duplicate the items in the inventory!

**Comment by Leo3418** [comment]
Thanks for reporting. Unfortunately, you did not provide sufficient information that could help us pinpoint or solve this issue. Please provide more details to your issue, such as your platform information, any error message, plugins that you have installed, steps to replicate the issue, etc. Otherwise, we may close this issue.

**Comment by sc00scoc** [comment]
Genisys last version if i execute this code
    public function onDrop(PlayerDropItemEvent $event) {
        $player = $event->getPlayer();
        if ($player->getLevel()->getName() == "SpawnDef") {
            //$event->setCancelled(true);
        }
        if ($this->onXZFactionSpawn($player->getX(), $player->getZ())) {
            //$event->setCancelled(true);
        }
    }

**Comment by Leo3418** [comment]
I did not replicate this issue with the following code:
```php
<?php

namespace Test01;

use pocketmine\Player;
use pocketmine\event\Listener;
use pocketmine\player\PlayerDropItemEvent;
use pocketmine\plugin\PluginBase;

class Test01 extends PluginBase implements Listener{

	public function onDrop(PlayerDropItemEvent $event){
		$player = $event->getPlayer();
		if($player->getLevel()->getName() == "world"){
			$event->setCancelled(true);
		}
	}
}

?>


```

**Comment by Leo3418** [comment]
Please provide more details to your issue, such as your platform information, plugins that you have installed, etc.

**Comment by xISRAPILx** [comment]
Em... All work.
![image](https://user-images.githubusercontent.com/28113262/28991432-1754f768-7990-11e7-930e-76493f2d5d63.png)

**Comment by Leo3418** [comment]
![ezgif-2-fb7c883176](https://user-images.githubusercontent.com/14175175/28991455-82a4807c-79ba-11e7-9995-b6c7e07e9aa6.gif)

Source code of the plugin I wrote: 
`plugin.yml`
```yml
name: Test01
main: Test01\Test01
version: 1.0.0
api: 3.0.1
load: POSTWORLD
```
`src/Test01/Test01.php`
```php
<?php

namespace Test01;

use pocketmine\Player;
use pocketmine\event\Listener;
use pocketmine\event\player\PlayerDropItemEvent;
use pocketmine\plugin\PluginBase;

class Test01 extends PluginBase implements Listener{
	public function onEnable(){
		$this->getServer()->getPluginManager()->registerEvents($this, $this);
	}

	public function onDrop(PlayerDropItemEvent $event){
		$player = $event->getPlayer();
		if($player->getLevel()->getName() == "world"){
			$event->setCancelled(true);
		}
	}
}

?>
```

**Comment by Leo3418** [comment]
Please provide more details to your issue, such as your platform information, plugins that you have installed, etc.

**Comment by xISRAPILx** [comment]
ckeck #703  @Leo3418

**Comment by sc00scoc** [comment]
![untitled](https://user-images.githubusercontent.com/25847440/28994490-65aa14b8-79cf-11e7-8d1a-01fc905e119d.gif)

**Comment by sc00scoc** [comment]
![untitled 1](https://user-images.githubusercontent.com/25847440/28994517-f80498c4-79cf-11e7-8db5-60cab6724ad9.gif)

**Comment by sc00scoc** [comment]
32 + 32 + other diamond in inv

**Comment by Leo3418** [comment]
Your GIF's quality is too low to be clearly seen. Please upload another one with higher quality.

**Comment by sc00scoc** [comment]
https://www.youtube.com/watch?v=ZS_SjaVGAXA

**Comment by MasterOfTheRealm** [comment]
@sc00scoc do you have allow-cheats = true in genisys.yml?

**Comment by sc00scoc** [comment]
no, No, because if I had enabled I could not use inventory on win10
